### ISSUE TITLE
Exception retrieving user information: Array to String Conversion

### ISSUE THREAD

**Issue by kphillipsjr1101** [leskhq/Laravel-Enterprise-Starter-Kit#116]
I'm working on a project developing a local intranet for a client. I'm trying to test authentication against LDAP with our DC. Everytime it goes to connect to the adldap I get this exception. I've tried multiple configurations but don't seem to be getting any traction. Here is a snapshot of my .env - my .settings-development reflects the eloquent.ldap portion of the .env file exactly. I'm confused what's going wrong here.

`# APP_ENV
# Value set in '/etc/php5/fpm/php-fpm.conf'
# based on the value from 'Homestead.yaml'
APP_ENV=development
APP_DEBUG=true
APP_KEY=62v7CqWMPNA4FVbvRiA12sNSyyO2Lvez

SQL_LOG=true
SQL_LOG_REQUEST=true

DB_CONNECTION=sqlite
DB_HOST=localhost
DB_DATABASE=homestead
DB_USERNAME=homestead
DB_PASSWORD=secret

CACHE_DRIVER=file
SESSION_DRIVER=file
SESSION_EXPIRE_ON_CLOSE=false
QUEUE_DRIVER=sync

MAIL_DRIVER=sendmail
#MAIL_HOST=mailtrap.io
#MAIL_PORT=2525
#MAIL_USERNAME=null
#MAIL_PASSWORD=null
#MAIL_ENCRYPTION=null

mail.system_sender_address=system.noreply@lesk.io
mail.system_sender_label=LESK System

lern.enable_record=true
lern.enable_notify=true
lern.channel=LESK
lern.mail_recipient=lesk.admin@email.com

LDAP_ENABLED=true
eloquent-ldap.enabled=true
eloquent-ldap.debug=false
eloquent-ldap.server_type=LDAP
eloquent-ldap.create_accounts=true
eloquent-ldap.replicate_group_membership=false
eloquent-ldap.resync_on_login=false
eloquent-ldap.group_model=App\Models\Role
eloquent-ldap.label_internal=internal
eloquent-ldap.label_ldap=ldap
eloquent-ldap.account_suffix=
eloquent-ldap.base_dn=dc=****,dc=com
eloquent-ldap.server=btcdc.****.com
eloquent-ldap.port=389
eloquent-ldap.user_name=CN=btcReader,OU=BTUsers,DC=****,DC=com
eloquent-ldap.password=*****
eloquent-ldap.return_real_primary_group=true
eloquent-ldap.secured=false
eloquent-ldap.secured_port=636
eloquent-ldap.recursive_groups=true
eloquent-ldap.sso=false
eloquent-ldap.username_field=uid
eloquent-ldap.email_field=mail
eloquent-ldap.first_name_field=
eloquent-ldap.last_name_field=sn
eloquent-ldap.user_filter=(&(objectclass=person)(uid=%username))
`

**Comment by kphillipsjr1101** [comment]
I've left out some addresses for our domain and the passwords to show and figure this out.

**Comment by sroutier** [comment]
Closed ?!! Already, I was about to reply. Did you resolve the problem? Let us know what was the issue if you do not mind that might help others.

Thanks.
/s

**Comment by kphillipsjr1101** [comment]
I tried a few different things, but I'm fairly certain that the password I used for the ldap reader user was causing issues. Another thing I noticed was that editing .env files and the .settings-development file directly, outside of the webportal would cause issues. Importing older versions that we had made of the .blade files would also cause fatal exceptions, presumably because they had some sort of unique key in them but I could not decipher what was causing them.
